test_flight_paths.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+
+def test_extract_data_from_json():
+    """Test extracted data."""
+    from flight_paths import extract_data_from_json
+    data = extract_data_from_json('cities_with_airports.json')
+    assert len(data) > 0
+    assert type(data) is list
+    assert type(data[0]) is dict
+
+
+def test_build_graph():
+    """Assert graph is built properly."""
+    from flight_paths import build_graph
+    from flight_paths import extract_data_from_json
+    data = extract_data_from_json('cities_with_airports.json')
+    gr = build_graph(data)
+
+    assert len(gr.edges()) > 0
+    assert len(gr.node_dict) > 0
+    assert len(gr.nodes()) > 0
+    assert len(gr.neighbors(gr.node_dict['Charlotte'])) > 70
+
+
+def test_depth_first_search():
+    """Assert search was completed."""
+    from flight_paths import build_graph
+    from flight_paths import extract_data_from_json
+    from flight_paths import depth_first_search
+    data = extract_data_from_json('cities_with_airports.json')
+    gr = build_graph(data)
+    x = depth_first_search(gr, gr.node_dict['Charlotte'], gr.node_dict['Honolulu'])
+    assert x[0][0] == 'Charlotte' and x[0][-1] == 'Honolulu'
```

Comment:
What about a test for when there is no path between two cities?
What about a test that when there is a path between two cities with your depth-first search, definite connections are hit along the way?